app/code/Magento/CustomerGraphQl/Model/Customer/ValidateCustomerData.php

```diff
@@ -60,4 +76,26 @@ public function execute(array $customerData): void
             );
         }
     }
+
+    /**
+     * Validate if email address is valid
+     *
+     * In order to work the same as in admin panel, the patter for validation was selected from 'validate-email'
+     * function in app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js
+     *
+     * @param string $email
+     *
+     * @return bool
+     */
+    private function emailIsValid(string $email): bool
```

Comment:
This method is redundant.
Additional JS validation should be preformed on frontend.